tests/services/users/test_user_moderation.py

```diff
@@ -16,17 +16,24 @@
 
 import pytest
 from invenio_access.permissions import system_identity
-from invenio_cache.lock import CachedMutex
+from marshmallow import ValidationError
 
 from invenio_users_resources.proxies import current_actions_registry
 from invenio_users_resources.services.users.lock import ModerationMutex
 
 
+@pytest.fixture()
```

Comment:
Previously the service did not check if a user was already blocked, now that it checks the test code doesnt' work anymore because the same users is being blocked multiple times, so this was a quick way of fixing the test to work.